src/activity_item_handling.cpp

```diff
@@ -1266,7 +1266,7 @@ void activity_on_turn_move_loot( player_activity &, player &p )
             // if it is, we can skip such item, if not we move the item to correct pile
             // think empty bag on food pile, after you ate the content
             if( !mgr.has( id, src ) ) {
-                const auto &dest_set = mgr.get_near( id, abspos );
+                const auto &dest_set = mgr.get_near( id, abspos, 60, thisitem->typeId() );
```

Comment:
What's the `60` represent here?

Reply:
range of the get_near search, in my farming PR im using 60 everywhere so that NPCs at the faction camp centre can find the farm zones that are more than the default 30 tiles away.

This will be preparing for consistency for when that merges aswell.

Reply:
In which case, do we want to put that into a named constant? That would both be self-documenting, and make it easier to change or use that distance in any other code that needs it.